docs/IR.md

```diff
@@ -351,7 +351,7 @@ Dimension variables are scoped to the declaration that they appear in. For graph
 
 For example, a graph that performs matrix cross-product may be defined as taking two inputs of shape [K,M] and [M,N], and producing an output of shape [K,N].
 
-The emptry string "", when used as a dimension name, denotes a single dimension of any cardinality. The string "*", when used as a dimension name, denotes zero or more dimensions of unknown cardinality.
+The empty string "", when used as a dimension name, denotes a single dimension of any cardinality. The string "*", when used as a dimension name, denotes zero or more dimensions of unknown cardinality.
```

Comment:
Hmmm … this (the original version itself, I don't mean this PR's change) is inaccurate. There were proposals for both these extensions, but I don't believe either is supported. I believe the statement should be entirely dropped.

Reply:
ok, i can remove it